python/semantic_kernel/plugin_definition/__init__.py

```diff
@@ -3,8 +3,11 @@
     kernel_function_context_parameter,
 )
 from semantic_kernel.plugin_definition.kernel_function_decorator import kernel_function
+from semantic_kernel.plugin_definition.kernel_plugin import KernelPlugin
 
 __all__ = [
     "kernel_function",
     "kernel_function_context_parameter",
+    "KernelPlugin",
+    "KernelPluginCollection" "DefaultKernelPlugin",
```

Comment:
I think this is missing a `,` after `"KernelPluginCollection"`.

Reply:
Good catch! Thanks.